packages/api-page-builder-so-ddb/src/definitions/blockEntity.ts

```diff
@@ -0,0 +1,58 @@
+import { Entity, Table } from "dynamodb-toolbox";
```

Comment:
Same here as in my previous comment. Let's be more consisten and name this `pageBlockEntity`. The same way we have it here:
<img width="637" alt="image" src="https://user-images.githubusercontent.com/5121148/171821184-ffb594c5-9fe0-4036-87e2-bbe7099f0fa6.png">

Reply:
answered in issue above
https://github.com/webiny/webiny-js/pull/2458/files#r891042674